recipes/dionysus/build.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+export CXX=clang++
```

Comment:
This should be in an `if [[ $(uname) == Linux ]]` or similar block. It happens to not mess stuff up on Mac apparently, but still better to stick to the default settings from `toolchain` if they work.